src/pages/home/release-notes/index.md

```diff
@@ -161,8 +179,8 @@ Keywords:
 
 ### Android Optimize 3.3.0
 
-* **[Breaking Change]** Updated Offer Score data type of from `Integer` to `Double`.
-* Optimize SDK only listens to preview events if it contains `debug` object with `eventType` as `com.adobe.eventType.edge` and `eventSource` as `personalization:decisions`.
+* **[Breaking Change]** Updated Offer Score data type of from `Integer` to `Double`.
```

Comment:
Same comment as above - it might be clearer if we just make this "Updated Offer Score data type from `Integer` to `Double`.

Reply:
Addressed here https://github.com/AdobeDocs/aep-mobile-sdkdocs/pull/753/commits/ec36a34aaa2adad9f64a02eb5c9173fc0931286d